core/templates/pages/welcome-guide-page/welcome-guide-page.directive.html

```diff
@@ -0,0 +1,79 @@
+<div style="height: auto; background-color: #FFFFFF;" class="row content">
```

Comment:
Add a class name here and move this style to the class (Ditto elsewhere in this file)

Reply:
added